Add volume 2 issue 2 (may 15) (#70)
techworkersco/techworkersco.github.io: _posts/2020-05-15-issue-2.md

---
layout: post
title: 'Issue 2: Who calls the shots?'
category: Volume 2
image:
    file: vol-2-issue-2-header.png
    alt: "Screenshot of a tweet sent by Elon Musk announcing the re-opening of a Tesla factory, contra local safety regulations"
    caption: "Elon Musk acting as if he's some sort of civil rights hero for defying regulations meant to keep workers safe"
    source_link: "https://twitter.com/elonmusk/status/1259945593805221891"
    half_width: false
preview: "Elon Musk gets to call the shots. He shouldn't."
---

> Who calls the shots at Tesla Motors? Is it the workers who collectively build all the products that generate the company's wealth? No, it's the company's multi-billionaire CEO, who recently threw a fit on Twitter over health and safety regulations that would delay resuming production. How does he manage to get away with that?

Reviewer: minor clarification to highlight the cause of Musk's tantrum: "who recently threw a fit on Twitter over COVID-19 shelter-in-place orders that would delay resuming production."

Reviewer 2: done

<!--excerpt-->

{% include post_image.html %}

Tesla CEO Elon Musk has done a great job of framing his decision to resume production as if it were a noble act on behalf of his workers. But [leaked emails from management](https://www.theguardian.com/technology/2020/may/12/tesla-workers-unemployment-benefits-emails) show that Tesla workers are being intimidated to return to work with the claim that they may be ineligible for unemployment benefits if they do not return to work. This isn't Musk taking a heroic stand to save his workers; it's a means of exercising the power that he holds over his workers during a global pandemic, so that they can keep making him money even if it puts their lives at risk.

> We’ve been here before. Jose Moran, a Tesla quality inspector at the plant in Fremont California, saw coworkers put in 60-70 hours each week and get injured on the job. In 2016, he reached out to the United Auto Workers for help, and in 2017 he wrote [an open letter](https://medium.com/@moran2017j/time-for-tesla-to-listen-ab5c6259fc88): “Most Tesla production workers earn between $17 and $21 hourly. The average auto worker in the nation earns $25.58 an hour, and lives in a much less expensive region.” Unfortunately, within two years, the unionizing campaign had failed. A ruling by the National Labor Relations Board (NLRB) indicated that [Tesla had illegally sabotaged organizing efforts](https://www.theverge.com/2018/9/29/17914572/tesla-union-trial-silicon-valley-unionization-elon-musk) with allegations that a union would render workers voiceless.

Reviewer 3: Missing comma between "Fremont" and "California".

Reviewer 2: fixed

> We've been here before. Jose Moran, a Tesla quality inspector at the plant in Fremont, California, saw coworkers put in 60-70 hours each week and get injured on the job. In 2016, he reached out to the United Auto Workers for help, and in 2017 he wrote [an open letter](https://medium.com/@moran2017j/time-for-tesla-to-listen-ab5c6259fc88): "Most Tesla production workers earn between $17 and $21 hourly. The average auto worker in the nation earns $25.58 an hour, and lives in a much less expensive region." Unfortunately, within two years, the unionizing campaign had failed. A ruling by the National Labor Relations Board (NLRB) indicated that [Tesla had illegally sabotaged organizing efforts](https://www.theverge.com/2018/9/29/17914572/tesla-union-trial-silicon-valley-unionization-elon-musk) with allegations that a union would render workers voiceless.

Reviewer: ```suggestion
We've been here before. Jose Moran, a Tesla quality inspector at the plant in Fremont, California, saw coworkers put in 60-70 hours each week and get injured on the job. In 2016, he reached out to the United Auto Workers for help, and in 2017 he wrote [an open letter](https://medium.com/@moran2017j/time-for-tesla-to-listen-ab5c6259fc88): "Most Tesla production workers earn between $17 and $21 hourly. The average auto worker in the nation earns $25.58 an hour, and lives in a much less expensive region." Unfortunately, within two years, the unionizing campaign had failed. A ruling by the National Labor Relations Board (NLRB) indicated that [Tesla had illegally sabotaged organizing efforts](https://www.theverge.com/2018/9/29/17914572/tesla-union-trial-silicon-valley-unionization-elon-musk) with allegations that Musk threatened by joining a union, "you don’t really have a voice."
```

Reviewer 2: hm, i find the flow of edit wording a little weird. i'll tweak this a bit.

[California is suing Uber and Lyft](https://www.nytimes.com/2020/05/05/technology/california-uber-lyft-lawsuit.html) for misclassifying drivers as independent contractors rather than employees under [AB-5](https://en.wikipedia.org/wiki/California_Assembly_Bill_5_(2019)), a new state law that took effect on January 1, 2020. This is a significant win for workers and organizing groups like [Rideshare Drivers United](https://drivers-united.org) who have been advocating for its implementation and enforcement. Rideshare drivers are tech workers, too. If you are a driver, [you can file a wage claim here](https://drivers-united.org/a/submit-a-wage-claim).

> In Chicago, Instacart shopper and organizer Matthew Telles of [Gig Workers Collective](https://www.gigworkerscollective.org) gives a personal look into the deteriorating working conditions that prompted gig workers to fight back in [an interview for Picture Theory](https://podcasts.apple.com/us/podcast/matthew-telles-gig-workers-are-standing-up-to-big-tech/id1361867735?i=1000473489342).

Reviewer: Add more at the end of this paragraph to quote Matthew, and to match the size of the other paragraphs:

“Grocery workers are dying and Instacart isn’t doing them any favors,” says Matthew. He also talks about how Instacart customers lure shoppers with "tip baiting," how pay and pretty much everything bottom-out after the first 90 days of being a shopper, and the unique challenges of building solidarity among gig workers and independent contractors.

Reviewer 2: done

> [Hate groups promoting violence](https://www.metrotimes.com/news-hits/archives/2020/05/11/whitmer-becomes-target-of-dozens-of-threats-on-private-facebook-groups-ahead-of-armed-rally-in-lansing) are continuing to flourish in private Facebook groups. *“Dozens of angry Michiganders, fueled by conspiracy theories and disinformation about the coronavirus, are promoting violence and mobilizing armed rallies against Gov. Gretchen Whitmer on Facebook, in violation of the social media company’s policies.”* Facebook’s inaction and complacency are unacceptable during this unprecedented public health crisis. Meanwhile, 11,250 current and former content moderators working for Facebook globally &mdash; from [Phoenix](https://www.theverge.com/2019/2/25/18229714/cognizant-facebook-content-moderator-interviews-trauma-working-conditions-arizona) to [the Philippines](https://www.vice.com/en_us/article/ywe7gb/the-companies-cleaning-the-deepest-darkest-parts-of-social-media) &mdash; were awarded [a $52M settlement](https://www.theverge.com/2020/5/12/21255870/facebook-content-moderator-settlement-scola-ptsd-mental-health) of at least $1,000 each. This is a major symbolic victory but a minor cost to Facebook, and a very small amount to compensate for what may be life-long post-traumatic stress for handling some of the worst content on the web.

Reviewer 3: Not sure about the intent of the part that's bolded (that starts "Dozens of angry Michiganders..."). Is it a quote from the linked article? Quotes in other sections seem to be emphasized with italics, using `_` instead of `*`, so that seems inconsistent.

Reviewer 2: Ah yes, single `_` and `*` are italics, double are bold.

Reviewer 2: i'll switch to use `_`

> [Google is rolling back its diversity & inclusion initiatives](https://www.nbcnews.com/news/us-news/current-ex-employees-allege-google-drastically-rolled-back-diversity-inclusion-n1206181) in order to avoid conservative backlash, allege current and former employees. A Google representative claims that it's because Google doesn't have "the global resources" to build that capacity internally, which seems a little hard to believe for a company with a [$100 billion cash hoard](https://www.theverge.com/2019/8/1/20749831/alphabet-google-apple-cash-reserves-richest-company) &mdash; not to mention a pattern of [retaliation against workers](https://www.vox.com/recode/2019/9/9/20853647/google-employee-retaliation-harassment-me-too-exclusive) who are reporting harassment. The tech industry at large has a long history with a severe lack of representation, not only in leadership roles but also in rank-and-file positions. We, as workers, cannot be complacent in this.

Reviewer: ```suggestion
[Google is rolling back its diversity & inclusion initiatives](https://www.nbcnews.com/news/us-news/current-ex-employees-allege-google-drastically-rolled-back-diversity-inclusion-n1206181) in order to avoid conservative backlash, allege current and former employees. A Google representative claims that it's because Google doesn't have "the global resources" to build that capacity internally, which seems a little hard to believe for a company with a [$100 billion cash hoard](https://www.theverge.com/2019/8/1/20749831/alphabet-google-apple-cash-reserves-richest-company) &mdash; not to mention a pattern of [retaliation against workers](https://www.vox.com/recode/2019/9/9/20853647/google-employee-retaliation-harassment-me-too-exclusive) who are reporting harassment. The tech industry at-large has a long history with a severe lack of representation, not only in leadership roles but also in rank-and-file positions. We, as workers, cannot be complacent in this.
```

Reviewer: at large => at-large

## In History

> From [collectiveactions.tech](https://collectiveactions.tech):

Reviewer: I think it's important to clarify that the Alliance@IBM was something we learned about from the collectiveactions.tech crew, but not something we're quoting from the website. I suggest that we remove the quotes below (the text is more legible without > styling), and add edit this line to something like, "Our friends from [collectiveactions.tech](https://collectiveactions.tech), a tech worker organizing database, pointed us to a story of the campaign to unionize IBM:"

Reviewer 2: > Our friends [...] pointed us to a story of the campaign to unionize IBM

Yes, I edited this in the google doc because it didn't feel genuine to me. 

They were not consulted on this, I just looked and chose a relevant action to highlight. I don't want to suggest they were actively involved when they weren't. I only want to provide some basic attribution, since we didn't discovered this ourselves.

> > On October 1, 1999 [the Communications Workers of America renewed their drive to organize IBM employees](https://cwa-union.org/news/entry/ibm_employees_and_cwa_launch_allianceibm). The CWA previously attempted to [unionize IBM in 1985](https://www.nytimes.com/1985/07/17/us/union-plans-drive-to-organize-ibm.html). This time around, the union started a new organization called Alliance@IBM. It reported that roughly 100,000 of the company's 140,000 workers were eligible for unionization. Union supporters started outreach at dozens of locations over the next 17 years, raising dozens of problems we see in tech today. Linda Croft, an employee in Endicott, NY, noted that IBM "will lay people off, then rehire them as contract or temporary workers &mdash; at much lower salaries and no benefits &mdash; for the same positions." The Alliance@IBM organizing ended in 2016. At its peak, it had 400 dues-paying members. Their website is [still viewable at archive.org](https://web.archive.org/web/20060104004355/http://www.allianceibm.org/) and [Watching IBM Facebook page](www.facebook.com/alliancemember) is still active, too.

Reviewer: ```suggestion
On October 1, 1999 [the Communications Workers of America renewed their drive to organize IBM employees](https://cwa-union.org/news/entry/ibm_employees_and_cwa_launch_allianceibm). The CWA previously attempted to [unionize IBM in 1985](https://www.nytimes.com/1985/07/17/us/union-plans-drive-to-organize-ibm.html). This time around, the union started a new organization called Alliance@IBM. It reported that roughly 100,000 of the company's 140,000 workers were eligible for unionization. Union supporters started outreach at dozens of locations over the next 17 years, raising dozens of problems we see in tech today. Linda Croft, an employee in Endicott, NY, noted that IBM "will lay people off, then rehire them as contract or temporary workers &mdash; at much lower salaries and no benefits &mdash; for the same positions." The Alliance@IBM organizing ended in 2016. At its peak, it had 400 dues-paying members. Their website is [still viewable at archive.org](https://web.archive.org/web/20060104004355/http://www.allianceibm.org/) and [their Facebook page](www.facebook.com/alliancemember) is still active, too.
```

Reviewer: minor grammatical edit re. the FB page
